test/gtest/uct/cuda/test_cuda_ipc_md.cc

```diff
@@ -193,6 +193,15 @@ UCS_TEST_P(test_cuda_ipc_md, mkey_pack_legacy)
 UCS_TEST_P(test_cuda_ipc_md, mkey_pack_mempool)
 {
 #if HAVE_CUDA_FABRIC
+    int driver_version;
+    EXPECT_EQ(CUDA_SUCCESS, cuDriverGetVersion(&driver_version));
```

Comment:
why check this way?
`Returns the latest version of CUDA supported by the driver.`
maybe use `cudaRuntimeGetVersion`, since this is a test?
or something like
```
cuDeviceGetAttribute(&major, CU_DEVICE_ATTRIBUTE_COMPUTE_CAPABILITY_MAJOR, dev);
    cuDeviceGetAttribute(&minor, CU_DEVICE_ATTRIBUTE_COMPUTE_CAPABILITY_MINOR, dev);
```

Reply:
The issue is in the driver, so we need to check the driver version, not the runtime version.